### PR TITLE
Updated SE list of responsibilities

### PR DESCRIPTION
**Pull request by mirie**
This is pulled from: https://docs.google.com/spreadsheets/d/1c0Ags_6-aJEEBuMC46pw5hshsmOD20IKuhPjigBbcSM/edit#gid=1241753057

### CODE REVIEW

File: Positions/SoftwareEngineer.md [hunk 1]
```diff
@@ -1,40 +1,42 @@
...
+- Comfortable designing data structures and schemas
+- Proficient with SQL
+
+### Impact
+- Capable of prioritizing tasks and makes progress without significant feedback from more senior engineers
+- Knows when to ask for help in order to get themselves unblocked and avoids getting caught up in unimportant details
+- Gaining experience in breaking down tasks, planning, and cutting unnecessary scope to achieve goals within a reasonable timeframe
+- Able to effectively estimate work
+- Comfortable making performance, stability, scalability, monitoring, and DevOps improvements
+- Self-sufficient in at least one large area of the codebase with a high level understanding of other components in the system"
```
Comment (mshmsh5000) [inline Positions/SoftwareEngineer.md]: Remove quote

File: Positions/SoftwareEngineer.md [hunk 2]
```diff
@@ -1,40 +1,42 @@
...
+
+### Communication
+- Clearly and effectively communicates with colleagues (across all mediums: PRs, Slack, email, in-person/video chat, etc)
+- Can clearly communicate technical information in a non technical manner
```
Comment (mshmsh5000) [inline Positions/SoftwareEngineer.md]: "non technical" -> "non-technical"

File: Positions/SoftwareEngineer.md [hunk 3]
```diff
@@ -1,40 +1,42 @@
...
+- Gives timely, helpful feedback to peers and offers guidance in PRs 
+- Asks productive questions to thoroughly understand requirements / user stories
+- Can identify contradictory requirements and suggest resolutions
+- Creates clear and concise sets of documentation around projects worked on for technical and non-technical audiences.
```
Comment (mshmsh5000) [inline Positions/SoftwareEngineer.md]: Remove ending period

File: Positions/SoftwareEngineer.md [hunk 4]
```diff
@@ -1,40 +1,42 @@
...
+
+### Leadership
+- Comfortable with sharing knowledge with other engineers (for example, at Dev Round Tables)
```
Comment (mshmsh5000) [inline Positions/SoftwareEngineer.md]: I've been inconsistent on "round  table" / "roundtable" myself, but given that this is a public doc, I think it's time for us to standardize. [Merriam Webster has _roundtable_](https://www.merriam-webster.com/dictionary/roundtable), so my suggestion here is

>...at Dev Roundtables

File: Positions/SoftwareEngineer.md [hunk 5]
```diff
@@ -1,40 +1,42 @@
...
+- Can conceive of and implement significant improvements to the codebase"
```
Comment (mshmsh5000) [inline Positions/SoftwareEngineer.md]: Remove trailing quote

File: Positions/SoftwareEngineer.md (hunk 2 above)
Comment (mirie) [reply Positions/SoftwareEngineer.md]: Updated in https://docs.google.com/spreadsheets/u/1/d/1c0Ags_6-aJEEBuMC46pw5hshsmOD20IKuhPjigBbcSM/edit#gid=1241753057 as well

File: Positions/SoftwareEngineer.md (hunk 4 above)
Comment (mirie) [reply Positions/SoftwareEngineer.md]: Updated in https://docs.google.com/spreadsheets/u/1/d/1c0Ags_6-aJEEBuMC46pw5hshsmOD20IKuhPjigBbcSM/edit#gid=1241753057 as well

**Comment by mirie** [comment]
@mshmsh5000 updated PR with changes! Thanks for those great 👀 !!!

**Pull request merged**